protoutil/blockutils.go

```diff
@@ -147,7 +155,7 @@ func GetConsenterMetadataFromBlock(block *cb.Block) (*cb.Metadata, error) {
 	res := &cb.Metadata{}
 	err = proto.Unmarshal(obm.ConsenterMetadata, res)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to unmarshal consenter metadata")
+		return nil, errors.Wrap(err, "failed to unmarshal consenter metadata")
```

Comment:
Same here

Reply:
I think this one is fine as-is since it's wrapping an error from the proto package instead of a fabric package that already generated the stack. I did an audit (just after this review comment came in) of the other errors and updated at least one other place where it was wrapping instead of withmessaging.